### ISSUE TITLE
Deploying to production

### ISSUE THREAD

**Issue by tslawecki** [glos/myglos#246]
@kknee, @gitchrisadams & @bobfrat, please advise what needs to done to move this to production - add vCPUs + RAM + disk to myglos server, or set up a clone? @ColinNBrooks has been waiting patiently since mid-August to get back to his client with production URL and new user IDs and passwords.

**Comment by benjwadams** [comment]
In terms of specs, I'd say
  \>= 100 GB HD space
 6-8 GB RAM
 4 vCPU

**Comment by tslawecki** [comment]
So just set up a centos VM and make that available to RPS?

**Comment by benjwadams** [comment]
So long as it's CentOS 7, that should be fine.

**Comment by tslawecki** [comment]
Setting it up now. I think it will be at 192.168.76.37 (internal) and 64.9.201.77 (external). Anything needed for external access beyond http/https?

**Comment by tslawecki** [comment]
We have a VM up now at 192.168.76.37, but the tech had to move to some other urgent items. @benjwadams, LMK if you'd like me to send you the root PW. I would encrypt and send by e-mail, then text you the encryption key.

**Comment by gitchrisadams** [comment]
@tslawecki I'm working with Ben on getting this setup. Can you send the encrypted root PW info to christopher.adams@rpsgroup.com Thanks.

**Comment by gitchrisadams** [comment]
@tslawecki @benjwadams and I were able to get thing setup on the server. Can you setup DNS to point to whatever link you want for production oil spill? Also is an https cert necessary? 
cc: @Bobfrat

**Comment by tslawecki** [comment]
https is necessary. There is a glos.us cert, what pieces should I be looking for?

**Comment by gitchrisadams** [comment]
@tslawecki Just saw your email but just wanted to update issue as well. https://hecspill.glos.us should now be accessible.

**Comment by ColinNBrooks** [comment]
Do I need to create a new account that can access this site? I just tried to log on with cnbrooks@mtu.edu but there is no such user; same with the wtpgrouptesting@gmail.com account we were using on the dev version.

**Comment by gitchrisadams** [comment]
@ColinNBrooks Yes, I believe you will need to create a new account since this is a new instance. Let me know if you have any issues with that.

**Comment by ColinNBrooks** [comment]
Thanks. I went to https://hecspill.glos.us/user/new & was able to create a new account. When I log on I can now see the HEC spills data including the sensitive data tied to the water intakes. @tslawecki - we should probably have some kind of approval process for new accounts that can see the water intake spill data, since we agreed to restrict access to only approved users. Is that something we can do?

![image](https://user-images.githubusercontent.com/20344813/46302967-9452c580-c578-11e8-8b16-5377dd98ccf8.png)
